app/views/shared/_document_overview_list.html.erb

```diff
@@ -57,16 +57,22 @@
             <li>
               <%= link_to document.upload.filename, portal_document_path(id: document.id), target: "_blank", rel: "noopener noreferrer" %>
             </li>
+            <% if documents.length <= 1 && current_intake.filing_joint == "yes" %>
```

Comment:
we tend to prefer `current_intake.filing_jointly?` instead of `current_intake.filing_joint == "yes"`